### ISSUE TITLE
[beta 4] [BUG] Unwanted Retract at beginning of a print 

### ISSUE THREAD

**Issue by hevilp** [Ultimaker/CuraEngine#990]
**Application Version**

**Platform**

**Display Driver**

**Steps to Reproduce**
Slicing an object and retract in the beginning, which I don't want and not configured

G1 F3600 E-4 is unwanted and I'm unable to disable,

**Actual Results**
```
;FLAVOR:Marlin
;TIME:454
;Filament used: 0.273263m
;Layer height: 0.2
;Generated with Cura_SteamEngine 4.0.0-BETA
M82 ;absolute extrusion mode
;start_gcode EXTRUDER=240 BED=100
G92 E0
G1 F3600 E-4
;LAYER_COUNT:2
;LAYER:0
M107
```

**Comment by Ghostkeeper** [comment]
Which printer are you using? This looks like it's in your start g-code.

**Comment by hevilp** [comment]
No,

my startgcode is the line with start_gcode EXTRUDER=240 BED=100

I'm on a Voron V2.1

**Comment by Ghostkeeper** [comment]
Ah, maybe you could attach a project file then so that we can reproduce the problem.

**Comment by hevilp** [comment]
3mf file?

[V2_fsr_cali.zip](https://github.com/Ultimaker/CuraEngine/files/2841402/V2_fsr_cali.zip)

**Comment by Ghostkeeper** [comment]
No, a project file. That is File -> Save...

This produces a 3MF file that also contains all of your settings.

**Comment by hevilp** [comment]
[V2_z_drive_rear_support_b_5mm_x2.zip](https://github.com/Ultimaker/CuraEngine/files/2841421/V2_z_drive_rear_support_b_5mm_x2.zip)

**Comment by Ghostkeeper** [comment]
Oh, you're not using a machine definition that is default in Cura. I thought you might be using Custom FFF Printer, but it seems you have a different distribution of Cura than the one we develop, or you modded it in some way? Where did you get your definition for Voron_2_350?

**Comment by hevilp** [comment]
No, it is a default Cura Version with ressources files from the Voron community

**Comment by hevilp** [comment]
Where does the 4 mm does come from? I got 5 mm retract in my config, not 4 mm

**Comment by Ghostkeeper** [comment]
I don't know, but a start code with only a comment line, and a retraction that doesn't use the normal retraction length sounds really suspicious to me :stuck_out_tongue: 

Where did you get those resource files for your Cura mod then? I found a Reddit thread but no definition files there.

**Comment by hevilp** [comment]
In the discord community, but I checked the files, there is nothing with the retraction in it.

BTW: my whole start code is in klipper ;) so I only need 1 line in every slicer....

**Comment by hevilp** [comment]
I use my own Profile, even here is a retract in the beginning...
